src/diffpy/utils/diffraction_objects.py

```diff
@@ -92,18 +92,18 @@ def __init__(
             The dependent variable array corresponding to intensity values.
         xtype : str
             The type of the independent variable in `xarray`. Must be one of {*XQUANTITIES}.
-        wavelength : float, optional
-            The wavelength of the incoming beam, specified in angstroms (Å). Default is none.
-        scat_quantity : str, optional
-            The type of scattering experiment (e.g., "x-ray", "neutron"). Default is an empty string "".
-        name : str, optional
-            The name or label for the scattering data. Default is an empty string "".
-        metadata : dict, optional
-            The additional metadata associated with the diffraction object. Default is {}.
+        wavelength : float, optional, default is None.
```

Comment:
With this format, `x, y, z.` it renders nicely

<img width="496" alt="Screenshot 2024-12-30 at 10 05 55 PM" src="https://github.com/user-attachments/assets/3108c293-1f9a-4e8c-a2ba-ca750ad4b76a" />